webpack.config.js

```diff
@@ -188,7 +189,33 @@ const webpackConfig = ( mode ) => {
 				maxEntrypointSize: 175000,
 			},
 			module: {
-				rules,
+				rules: [
+					...rules,
+					{
+						test: /\.(png|jpg)$/i,
+						use: [
+							{
+								loader: 'file-loader',
+								options: {
+									name: '[name].[ext]',
```

Comment:
Since we're loading these via webpack now, can we add a hash to the filename as well? While not as critical as JS, they face the same potential browser caching problems. It seems like this would be an easy change, no?

Reply:
No, we can't add hashes because it will break images in the storybook and VRT tests.

Reply:
I understand changes in storybook could break VRT, but changing the filename should have no affect on VRT by itself.

I think the only thing we would need to do is update `stories/notifications.stories.js` to import the images as well like they are in the app now, no?

It's not a big deal, but just trying to understand the obstacles to doing this, and we should try to keep our use in Storybook consistent with the app.

Reply:
I haven't looked at it too deeply but storybook puts all images into a different folder. So it will require more time to figure out how to configure it correctly. If you still want it to be implemented, I would suggest creating a new ticket for it.